src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -130,8 +162,29 @@ public void initGui()
 		}
 
 		this.buttonList.add( this.cancel );
+
+		this.terminalStyleBox = new GuiImgButton( this.guiLeft - 18, offset, Settings.TERMINAL_STYLE, AEConfig.instance().getConfigManager().getSetting( Settings.TERMINAL_STYLE ) );
+		this.buttonList.add( this.terminalStyleBox );
 	}
 
+	private void calculateRows( final int height )
+	{
+		final int maxRows = AEConfig.instance().getConfigManager().getSetting( Settings.TERMINAL_STYLE ) == TerminalStyle.SMALL ? 5 : Integer.MAX_VALUE;
+
+		final double extraSpace = this.height - height;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Cast one of the operands of this substraction operation to a "double". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2184)